plugins/chat/config/locales/client.en.yml

```diff
@@ -51,6 +51,7 @@ en:
       cancel_reply: "Cancel reply"
       chat_channels: "Channels"
       browse_all_channels: "Browse all channels"
+      see_in: "See in"
```

Comment:
This is not ideal as it will be hard for translators to know what we will see in... but channel is a component here as we render other things like the icon.

Reply:
Hey @jjaffeux, we can also use `# comments` that show up as context for the translators.

E.g.
https://github.com/discourse/discourse-assign/blob/ae26c05db59fb85ca0132cb52f4dbf6527cb361a/config/locales/client.en.yml#L37-L38

And on [Crowdin](https://discourse.crowdin.com/editor/10/312/en-fr/1212?view=comfortable&filter=basic&value=0#q=assign_post_to_multiple)

<img width="786" alt="image" src="https://github.com/discourse/discourse/assets/3530/a8fb01be-2a13-4162-aec0-90adddb1cba5">